Link Checker Report

# Summary

| Status         | Count |
|----------------|-------|
| 🔍 Total       | 1876  |
| ✅ Successful  | 1718  |
| ⏳ Timeouts    | 1     |
| 🔀 Redirected  | 0     |
| 👻 Excluded    | 135   |
| ❓ Unknown     | 0     |
| 🚫 Errors      | 22    |
| ⛔ Unsupported | 0     |

## Errors per input

### Errors in https://plan.research-data.no/pages/3_storage_backup.html

* [404] <https://cms.sikt.no/sites/default/files/2023-02/Sektorstandard_klassifisering_av_informasjon%20%281%29.pdf> | Rejected status code (this depends on your "accept" configuration): Not Found

### Errors in https://plan.research-data.no/pages/4_legal_ethics.html

* [300] <http://www.lovdata.no/all/nl-19921204-132.html> | Rejected status code (this depends on your "accept" configuration): Multiple Choices
* [300] <http://www.lovdata.no/all/nl-19990702-063.html> | Rejected status code (this depends on your "accept" configuration): Multiple Choices
* [300] <http://www.lovdata.no/all/nl-19990702-064.html> | Rejected status code (this depends on your "accept" configuration): Multiple Choices
* [300] <http://www.lovdata.no/for/sf/ho/ho-20090701-0955.html> | Rejected status code (this depends on your "accept" configuration): Multiple Choices
* [300] <http://www.lovdata.no/for/sf/ho/ho-20091030-1321.html> | Rejected status code (this depends on your "accept" configuration): Multiple Choices
* [302] <https://www.ga4gh.org/genomic-data-toolkit/regulatory-ethics-toolkit/> | Rejected status code (this depends on your "accept" configuration): Found
* [TIMEOUT] <https://www.gnu.org/licenses/gpl-3.0.en.html> | Timeout

### Errors in https://plan.research-data.no/pages/difficult_faq.html

* [403] <https://doi.org/10.1093/gigascience/giad049> | Rejected status code (this depends on your "accept" configuration): Forbidden
* [404] <https://www.youtube.com/watch?v=EiJ8RaD3WBw> | Rejected status code (this depends on your "accept" configuration): Not Found

### Errors in https://plan.research-data.no/pages/dmp_training.html

* [401] <https://dmeg.cessda.eu/Archived/Training/Training-Resources/Library/Training-Packages/Train-the-Trainers> | Rejected status code (this depends on your "accept" configuration): Unauthorized

### Errors in https://plan.research-data.no/pages/support_00_local_disc.html

* [403] <https://doi.org/10.7551/mitpress/12200.001.0001> | Rejected status code (this depends on your "accept" configuration): Forbidden

### Errors in https://plan.research-data.no/pages/support_03_legal_ethics.html

* [404] <https://book.the-turing-way.org/project-design/sdpw> | Rejected status code (this depends on your "accept" configuration): Not Found
* [403] <https://www.coe.int/en/web/bioethics/oviedo-convention> | Rejected status code (this depends on your "accept" configuration): Forbidden
* [404] <https://www.forskningsetikk.no/globalassets/dokumenter/4-publikasjoner-som-pdf/redusert-samtykkekompetanse-i-helsefaglig-forskning-2005.pdf> | Rejected status code (this depends on your "accept" configuration): Not Found
* [302] <https://www.ga4gh.org/genomic-data-toolkit/regulatory-ethics-toolkit/> | Rejected status code (this depends on your "accept" configuration): Found

### Errors in https://plan.research-data.no/pages/support_05_reuse_collect.html

* [405] <https://app.dimensions.ai/> | Rejected status code (this depends on your "accept" configuration): Method Not Allowed
* [429] <https://commons.datacite.org/> | Rejected status code (this depends on your "accept" configuration): Too Many Requests
* [403] <https://doi.org/10.3233/DS-190026> | Error (cached)
* [403] <https://www.europeana.eu/en> | Rejected status code (this depends on your "accept" configuration): Forbidden

### Errors in https://plan.research-data.no/pages/support_06_document.html

* [403] <https://doi.org/10.1093/gigascience/giz165> | Rejected status code (this depends on your "accept" configuration): Forbidden
* [ERROR] <https://rdamsc.bath.ac.uk/> | Network error: error sending request for url (https://rdamsc.bath.ac.uk/) Maybe a certificate error?

### Errors in https://plan.research-data.no/pages/support_08_preserve_publish.html

* [403] <https://doi.org/10.3233/DS-190026> | Rejected status code (this depends on your "accept" configuration): Forbidden
[Full Github Actions output](https://github.com/DHP-stottepakke/pages/actions/runs/17837863285?check_suite_focus=true)

--- a/pages/dmp_knowledge_base/3_storage_backup.md
+++ b/pages/dmp_knowledge_base/3_storage_backup.md
@@ -232,7 +232,7 @@ Missing:
 ### Knowledge for support staff
 - National policies
 - Classification of information: Institutional policies and storage guides
-	- [Klassifisering av infomasjon: Sektorstandard for universiteter, høgskoler og forskningsinstitutter](https://cms.sikt.no/sites/default/files/2023-02/Sektorstandard_klassifisering_av_informasjon%20%281%29.pdf "2024-09-11")
+	- [Klassifisering av infomasjon: Sektorstandard for universiteter, høgskoler og forskningsinstitutter](https://cms.sikt.no/sites/default/files/2024-10/Veileder%20i%20ledelsessystem%20for%20informasjonssikkerhet.pdf "2025-09-19")
 	- Dual use research: Report [Et helhetlig forskningssystem for åpen, skjermet og gradert forskning (NFR, 2024)](https://www.forskningsradet.no/nyheter/2024/fire-anbefalinger-for-et-helhetlig-forskningssystem-for-apen-skjermet-og-gradert-forskning/ "2024-09-11")
 	- Export control: Guidelines for [International research and innovation cooperation](https://hkdir.no/en/guidelines-and-tools-for-responsible-international-knowledge-cooperation/international-research-and-innovation-cooperation "2024-10-03")
 - Routines for data recovery

--- a/pages/dmp_knowledge_base/4_legal_ethics.md
+++ b/pages/dmp_knowledge_base/4_legal_ethics.md
@@ -582,4 +582,4 @@ How will you manage the rights of the data you use, produce and share? (2.2) Des
 **Tips for best practices**
 - Agreements on rights of use should be made as early as possible in the project life cycle.
 - Have you gained consent for data preservation and sharing? • Follow the funder's or publisher's policies.
-- It is recommended to make all of the research data, code and software created within a research project available for reuse, e.g., under a [Creative Commons](https://creativecommons.org/choose/), [GNU](https://www.gnu.org/licenses/gpl-3.0.en.html) or [MIT license](https://opensource.org/licenses/MIT), or under another relevant license.
+- It is recommended to make all of the research data, code and software created within a research project available for reuse, e.g., under a [Creative Commons](https://creativecommons.org/choose/), [GNU](https://opensource.org/license/GPL-3.0) or [MIT license](https://opensource.org/licenses/MIT), or under another relevant license.

--- a/pages/dmp_knowledge_base/4_legal_ethics.md
+++ b/pages/dmp_knowledge_base/4_legal_ethics.md
@@ -56,7 +56,7 @@ According to GDPR there are two options for legal bases for processing of person
 
 Please note that the specific 'consent' under GDPR as a legal basis is not equivalent with 'informed consent' in the context of health research. For more information please consult the [Opinion 3/2019 concerning the Questions and Answers on the interplay between the Clinical Trials Regulation (CTR) and the General Data Protection regulation (GDPR)](https://edpb.europa.eu/our-work-tools/our-documents/opinion-art-70/opinion-32019-concerning-questions-and-answers_en) of the European data protection board.
 
-If consent is used as legal basis for processing of personal data consider asking for permission to archive the data for the purpose of future research, and possibly also educational purposes as part of both the informed and specific consent. It is advisable to use standard consent clauses, which can be reflected with machine readable metadata (See e.g. the [GA4GH ethical toolkit](https://www.ga4gh.org/genomic-data-toolkit/regulatory-ethics-toolkit/) as an example for human genetic data)
+If consent is used as legal basis for processing of personal data consider asking for permission to archive the data for the purpose of future research, and possibly also educational purposes as part of both the informed and specific consent. It is advisable to use standard consent clauses, which can be reflected with machine readable metadata (See e.g. the [GA4GH consent toolkit](https://www.ga4gh.org/product/consent-toolkit/) as an example for human genetic data)
 
 > 4a.3
 > * Consider anonymisation of personal data for preservation and/or sharing (truly anonymous data are no longer considered personal data).

--- a/pages/dmp_knowledge_base/difficult_faq.md
+++ b/pages/dmp_knowledge_base/difficult_faq.md
@@ -24,7 +24,7 @@ Science Europe, an organisation of European funders has created a [rubric for re
 
 In the Horizon Europe funding scheme, a brief outline of data management (1-page DMP) is required at proposal stage and included the scientific proposal evaluation.
 * [Horizon Europe - briefing slides for evaluators](https://ec.europa.eu/info/funding-tenders/opportunities/docs/2021-2027/experts/standard-briefing-slides-for-experts_he_en.pdf)
-* [Horizon Europe - video on evaluation of Open Science practices](https://ec.europa.eu/info/funding-tenders/opportunities/portal/screen/support/videos), [YouTube-link](https://www.youtube.com/watch?v=EiJ8RaD3WBw)
+* [Horizon Europe - video on evaluation of Open Science practices](https://ec.europa.eu/info/funding-tenders/opportunities/portal/screen/support/videos), 
 
 ### Where can I find an example of how a DMP should look like?
 Reading through different DMPs can be an useful exercise. Keep in mind that not all public DMPs are quality-controlled.

--- a/pages/dmp_knowledge_base/difficult_faq.md
+++ b/pages/dmp_knowledge_base/difficult_faq.md
@@ -24,7 +24,7 @@ Science Europe, an organisation of European funders has created a [rubric for re
 
 In the Horizon Europe funding scheme, a brief outline of data management (1-page DMP) is required at proposal stage and included the scientific proposal evaluation.
 * [Horizon Europe - briefing slides for evaluators](https://ec.europa.eu/info/funding-tenders/opportunities/docs/2021-2027/experts/standard-briefing-slides-for-experts_he_en.pdf)
-* [Horizon Europe - video on evaluation of Open Science practices](https://ec.europa.eu/info/funding-tenders/opportunities/portal/screen/support/videos), 
+* [Horizon Europe - video on evaluation of Open Science practices](https://ec.europa.eu/info/funding-tenders/opportunities/portal/screen/support/videos), [YouTube-link](https://youtu.be/v7rHdOn-JtU)
 
 ### Where can I find an example of how a DMP should look like?
 Reading through different DMPs can be an useful exercise. Keep in mind that not all public DMPs are quality-controlled.

--- a/pages/dmp_knowledge_base/dmp_training.md
+++ b/pages/dmp_knowledge_base/dmp_training.md
@@ -17,7 +17,7 @@ toc: true
 * [EIFL Digital Research Literacy Training Programme Outline for Librarians (2024): Writing a data management plan](https://eifl.net/programme/digital-research-literacy/writing-data-management-plan-dmp)
 * [FAIRsFAIR FAIR teaching handbook (2021): Lesson plan DMP](https://fairsfair.gitbook.io/fair-teaching-handbook/0lessonplans/2lessonplan)
 * [Data One: Data Management Planning Skillbuilding Hub (2020): Data Management Planning](https://dataoneorg.github.io/Education/lessons/03_planning/)
-* [CESSDA Train-the-trainers package (2020)](https://dmeg.cessda.eu/Archived/Training/Training-Resources/Library/Training-Packages/Train-the-Trainers) which is an addition to the Data management expert guide (DMEG)
+* [CESSDA Train-the-trainers package (2020)](https://www.cessda.eu/Training/Training-Resources/TTT) which is an addition to the Data management expert guide (DMEG)
 
 
 ## Useful RDM and FAIR resources

--- a/pages/dmp_supporting_info/support_03_legal_ethics.md
+++ b/pages/dmp_supporting_info/support_03_legal_ethics.md
@@ -28,7 +28,7 @@ turing:
 - name: Ethical Research
   url: https://book.the-turing-way.org/ethical-research/ethical-research
 - name: Sensitive Data Projects
-  url: https://book.the-turing-way.org/project-design/sdpw
+  url: https://book.the-turing-way.org/project-design/data-security/sdp
 ---
 
 {% include callout.html type="tip" content="
@@ -199,7 +199,6 @@ Please indicate any additional ethical guidelines or codes of conducts relevant 
 * [Ethics Guidelines for Trustworthy AI](https://digital-strategy.ec.europa.eu/en/library/ethics-guidelines-trustworthy-ai "2024-04-22")
 * [Nagoya Protocol on Access to Genetic Resources and Benefit-Sharing](https://www.cbd.int/abs/default.shtml "2024-04-22")
 * [Medical and health research in low- and middle-income countries by The National Committee for Medical and Health Research Ethics (NEM)](https://www.forskningsetikk.no/en/guidelines/medical-and-health-research/medical-and-health-research-in-low--and-middle-income-countries/ "2024-07-17")
-* [Guidelines for the inclusion of adults with impaired or absent capacity to consent *Norwegian only*](https://www.forskningsetikk.no/globalassets/dokumenter/4-publikasjoner-som-pdf/redusert-samtykkekompetanse-i-helsefaglig-forskning-2005.pdf)
 * [Payment to participants in medical or health research *Norwegian only*](https://www.forskningsetikk.no/retningslinjer/med-helse/betaling-til-deltakere-i-medisinsk-eller-helsefaglig-forskning/)
 * [Guidelines for the use of genetic studies of humans *Norwegian only*](https://www.forskningsetikk.no/retningslinjer/med-helse/retningslinjer-for-bruk-av-genetiske-undersokelser-av-mennesker-i-medisinsk-og-helsefaglig-forskning/)
 * [Ethical guidelines for clinical trial of drugs *Norwegian only*](https://www.forskningsetikk.no/ressurser/publikasjoner/kliniske-utprovinger-av-legemidler/)

--- a/pages/dmp_supporting_info/support_03_legal_ethics.md
+++ b/pages/dmp_supporting_info/support_03_legal_ethics.md
@@ -304,7 +304,7 @@ Institutional resources on participant information and consent:
 * [UiO - Open sharing: Research data and personal information](https://www.uio.no/english/for-employees/support/research/research-data-management/topics/as-open-as-possible.html)
 
 Further information:
-* [GA4GH ethical toolkit](https://www.ga4gh.org/genomic-data-toolkit/regulatory-ethics-toolkit/) with examples of granular consent and machine readable metadata (for human genetic data)
+* [GA4GH consent toolkit](https://www.ga4gh.org/product/consent-toolkit/) with examples of granular consent and machine readable metadata (for human genetic data)
 * [Informed Consent Ontology (ICO)](https://www.ebi.ac.uk/ols4/ontologies/ico)
 * [W3 Data Privacy Vocabulary (DPV)](https://w3c.github.io/dpv/)
 * For Arts and humanities you can get inspiration for consent forms from the [DARIAH ELDAH Consent Form Wizard (CFW)](https://consent.dariah.eu/) - make sure to clarify the form with your insitutional legal person as well

